packages/react/src/components/GroupedList/GroupedListV2.tsx

```diff
@@ -25,3 +25,4 @@ const GroupedListV2: React.FunctionComponent<IGroupedListV2Props> = styled<
 GroupedListV2.displayName = 'GroupedListV2_unstable';
 
 export { GroupedListV2 as GroupedListV2_unstable };
+export type { IGroupedListV2Props } from './GroupedListV2.types';
```

Comment:
Do you need to export it here if you're already exporting everything from `GroupedListV2.types` in the `index` file below?

Reply:
Probably not. However, I mirrored what is exported in `[GroupedList](https://github.com/microsoft/fluentui/blob/master/packages/react/src/components/GroupedList/GroupedList.ts#L15)` so I'm leaning toward having the export here but don't feel very strongly about it.